src/HtmlSanitizer/HtmlSanitizer.csproj

```
@@ -15,12 +18,12 @@
     <AssemblyOriginatorKeyFile>HtmlSanitizer.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
-    <PackageId>HtmlSanitizer</PackageId>
+    <PackageId>Stratsys.HtmlSanitizer.Fork</PackageId>
```

I och med att vi prefixa med STratsys känns det onödigt att postfixa med Fork. Tycker alltså det räcker med
```suggestion
    <PackageId>Stratsys.HtmlSanitizer</PackageId>
```